src/components/ebay-pagination/README.md

```diff
@@ -0,0 +1,56 @@
+# ebay-pagination
+
+The `<ebay-pagination>` is a tag used to create a pagination navigation.
+
+
+## ebay-pagination Tag
+
+### ebay-pagination Usage
+
+```marko
+<ebay-pagination aria-label-prev="Previous page" aria-label-next="Next page" curr-text="Results Pagination - Page 1">
+    <ebay-pagination-item href="#" previous disabled/>
+    <ebay-pagination-item href="#">item 1</ebay-pagination-item>
+    <ebay-pagination-item href="#" current>item 2</ebay-pagination-item>
+    <ebay-pagination-item href="#">item 3</ebay-pagination-item>
+    <ebay-pagination-item href="#" next/>
+</ebay-pagination>
+```
+
+### ebay-pagination Attributes
+
+Name | Type | Stateful | Description
+--- | --- | --- | ---
+`class` | String | No | custom class
+`aria-label-prev` | String | No | aria-label for previous button
+`aria-label-next` | String | No | aria-label for previous button
+`curr-text` | String | No | Description for the current page (e.g. Results of Page 1)
+`fake-link` | Boolean | No | Use pagination links for an ajax reload
```

Comment:
I like the purpose of this attribute (and breadcrumbs should be updated to match, in a separate PR), but we might need to choose another name for this. Maybe `type="spa"` or `type="hijax"` or something?

Reply:
I can have a boolean value as `hijax`. Having a `type="hijax"` isn't needed here as we don't have a 3rd value and we don't necessarily need a type for the default case.